apps/server/src/shared/testing/user-role-permissions.ts

```diff
@@ -5,297 +5,290 @@
+	Permission.SUBMISSIONS_EDIT,
+	Permission.FILESTORAGE_VIEW,
+	Permission.FILESTORAGE_EDIT,
+	Permission.FILESTORAGE_CREATE,
+	Permission.CONTENT_VIEW,
+	Permission.CONTENT_NON_OER_VIEW,
+	Permission.FILESTORAGE_REMOVE,
+	Permission.TEAM_VIEW,
+	Permission.CLASS_VIEW,
+	Permission.COURSE_VIEW,
+	Permission.LERNSTORE_VIEW,
 ] as Permission[];
 
 export const studentPermissions = [
-	'TASK_DASHBOARD_VIEW_V3',
-	'BASE_VIEW',
-	'DASHBOARD_VIEW',
-	'TOOL_VIEW',
-	'HOMEWORK_CREATE',
-	'PASSWORD_EDIT',
-	'FOLDER_DELETE',
-	'FOLDER_CREATE',
-	'FILE_DELETE',
-	'FILE_CREATE',
-	'FILE_MOVE',
-	'ACCOUNT_EDIT',
-	'CALENDAR_VIEW',
-	'CALENDAR_EDIT',
-	'CALENDAR_CREATE',
-	'FEDERALSTATE_VIEW',
-	'HELPDESK_CREATE',
-	'TOPIC_VIEW',
-	'LINK_CREATE',
-	'NEWS_VIEW',
-	'NOTIFICATION_VIEW',
-	'NOTIFICATION_EDIT',
-	'NOTIFICATION_CREATE',
-	'PWRECOVERY_VIEW',
-	'PWRECOVERY_EDIT',
-	'PWRECOVERY_CREATE',
-	'RELEASES_VIEW',
-	'ROLE_VIEW',
-	'COURSEGROUP_CREATE',
-	'COURSEGROUP_EDIT',
-	'SYSTEM_VIEW',
-	'HOMEWORK_VIEW',
-	'HOMEWORK_EDIT',
-	'COMMENTS_VIEW',
-	'COMMENTS_CREATE',
-	'COMMENTS_EDIT',
-	'SUBMISSIONS_VIEW',
-	'SUBMISSIONS_CREATE',
-	'SUBMISSIONS_EDIT',
-	'FILESTORAGE_VIEW',
-	'FILESTORAGE_EDIT',
-	'FILESTORAGE_CREATE',
-	'CONTENT_VIEW',
-	'CONTENT_NON_OER_VIEW',
-	'FILESTORAGE_REMOVE',
-	'TEAM_VIEW',
-	'TEAM_EDIT',
-	'TEAM_CREATE',
-	'PRIVACY_VIEW',
-	'CLASS_VIEW',
-	'COURSE_VIEW',
-	'LERNSTORE_VIEW',
-	'JOIN_MEETING',
-	'TOOL_CREATE_ETHERPAD',
+	Permission.TASK_DASHBOARD_VIEW_V3,
+	Permission.BASE_VIEW,
+	Permission.DASHBOARD_VIEW,
+	Permission.TOOL_VIEW,
+	Permission.HOMEWORK_CREATE,
+	Permission.PASSWORD_EDIT,
+	Permission.FOLDER_DELETE,
+	Permission.FOLDER_CREATE,
+	Permission.FILE_DELETE,
+	Permission.FILE_CREATE,
+	Permission.FILE_MOVE,
+	Permission.ACCOUNT_EDIT,
+	Permission.CALENDAR_VIEW,
+	Permission.CALENDAR_EDIT,
+	Permission.CALENDAR_CREATE,
+	Permission.FEDERALSTATE_VIEW,
+	Permission.HELPDESK_CREATE,
+	Permission.TOPIC_VIEW,
+	Permission.LINK_CREATE,
+	Permission.NEWS_VIEW,
+	Permission.NOTIFICATION_VIEW,
+	Permission.NOTIFICATION_EDIT,
+	Permission.NOTIFICATION_CREATE,
+	Permission.PWRECOVERY_VIEW,
+	Permission.PWRECOVERY_EDIT,
+	Permission.PWRECOVERY_CREATE,
+	Permission.RELEASES_VIEW,
+	Permission.ROLE_VIEW,
+	Permission.COURSEGROUP_CREATE,
+	Permission.COURSEGROUP_EDIT,
+	Permission.SYSTEM_VIEW,
+	Permission.HOMEWORK_VIEW,
+	Permission.HOMEWORK_EDIT,
+	Permission.COMMENTS_VIEW,
+	Permission.COMMENTS_CREATE,
+	Permission.COMMENTS_EDIT,
+	Permission.SUBMISSIONS_VIEW,
+	Permission.SUBMISSIONS_CREATE,
+	Permission.SUBMISSIONS_EDIT,
+	Permission.FILESTORAGE_VIEW,
+	Permission.FILESTORAGE_EDIT,
+	Permission.FILESTORAGE_CREATE,
+	Permission.CONTENT_VIEW,
+	Permission.CONTENT_NON_OER_VIEW,
+	Permission.FILESTORAGE_REMOVE,
+	Permission.TEAM_VIEW,
+	Permission.TEAM_EDIT,
+	Permission.TEAM_CREATE,
+	Permission.CLASS_VIEW,
+	Permission.COURSE_VIEW,
+	Permission.LERNSTORE_VIEW,
+	Permission.JOIN_MEETING,
+	Permission.TOOL_CREATE_ETHERPAD,
```

Comment:
I have fixed: "exclud duplications in seed-data". But this file has real duplicates for me. Can you please fix it:

```ts
export const studentPermissions = [ 
  ...userPermissions,
  Permission.TASK_DASHBOARD_VIEW_V3,
  ...
];

```